index.js

```diff
@@ -164,7 +167,7 @@ async function processItems(items, batchSize = 1) {
     // Calculate delay needed after each batch to adhere to the rate limit
     // Note: If batchSize is larger than the rate limit, this will result in a negative delay,
     // which should be handled as well (e.g., by setting a minimum batchSize or adjusting the logic accordingly).
-    const delayPerBatch = (60 / requestsPerMinute) * batchSize * 1000; // Convert to milliseconds
+    const delayPerBatch = 0 // Convert to milliseconds
```

Comment:
```suggestion
    const delayPerBatch = (60 / requestsPerMinute) * batchSize * 1000; // Convert to milliseconds
```

Can you leave the delay? Lately I am getting rate limited so I need that delay.

Reply:
Yeah I noticed that it throws 429, I changed it to do 30 per minute to see if it's enough for it to not rate limit

Reply:
sry, I thought you made the PR to my repository.

![image](https://github.com/jelolul/counter-strike-price-tracker/assets/38622893/c32f1c0a-fd4a-4b90-85d6-c0a4610141a5)

>Yeah I noticed that it throws 429, I changed it to do 30 per minute to see if it's enough for it to not rate limit

Interesting, seems like they've made changes recently. Couple of days ago I was able to make 23k request in <10 minutes.